### PR TITLE
auth chat multitab, widget load time improvement

### PR DESCRIPTION
**Pull request by sarojkpr**
- Fixing multitab issue when using `Microsoft.Omnichannel.LiveChatWidget.SDK.setAuthTokenProvider`
- Fixing post chat survey regression of not allowing to close chat widget if survey is not filled.
- Moving the postchat survey call after chat widget load to improve widget load time

Test URL : https://sacadlcw001.blob.core.windows.net/lcwtpc/Test/testAuth.html
Auth settings are configured like this in this HTML:
image

Screenshot:
Refresh scenario:
image

Duplicate tab:
image

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 1]
```diff
@@ -134,9 +135,6 @@ const initStartChat = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAct
                 }
             });
             isStartChatSuccessful = false;
-            // Resetting the widget state to Closed, for recent introduction of OC rate limiting(429 Error)
```
Comment (abhinab22mohanty) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:137]: Why are we removing this?

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 2]
```diff
@@ -269,43 +266,12 @@ const setCustomContextParams = (chatSDK: any) => {
     }
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (abhinab22mohanty) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:272]: Any reason we are removing this check?

File: chat-widget/src/components/livechatwidget/common/startChat.ts (hunk 1 above)
Comment (sarojkpr) [reply chat-widget/src/components/livechatwidget/common/startChat.ts:137]: To keep on loading pane on exception, not automatically show chat button

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 3]
```diff
@@ -269,43 +266,12 @@ const setCustomContextParams = (chatSDK: any) => {
...
-const handleAuthenticationIfEnabled = async (chatSDK: any, props: any): Promise<boolean> => {
```
[the remark this answers is not in the document]
Comment (sarojkpr) [reply chat-widget/src/components/livechatwidget/common/startChat.ts:273]: Its handled already inside start chat

**Review by charliewang95 — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 4]
```diff
@@ -236,8 +234,7 @@ const canConnectToExistingChat = async (props: ILiveChatWidgetProps, chatSDK: an
 
     //Connect to only active chat session
```
Comment (dishakhattri) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:235]: update the comment

**Review by dishakhattri — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/startChat.ts (hunk 4 above)
Comment (sarojkpr) [reply chat-widget/src/components/livechatwidget/common/startChat.ts:235]: will fix on next PR

**Pull request merged**